pkg/api/scheme/scheme.go

```diff
@@ -69,10 +89,17 @@ var builders = []runtime.SchemeBuilder{
 }
 
 func init() {
-	Scheme = runtime.NewScheme()
-	if err := addKnownTypes(Scheme); err != nil {
+	Scheme = &FilteredScheme{
+		Scheme:       runtime.NewScheme(),
+		excludedGVKs: make(map[schema.GroupVersionKind]bool),
```

Comment:
...oh and this would of course change:
```suggestion
		excludedGVKs: make(map[schema.GroupVersionKind]struct{}),
```